src/d3d9/d3d9_rtx.cpp

```diff
@@ -651,6 +651,11 @@ namespace dxvk {
     // Hash material data
     m_activeDrawCallState.materialData.updateCachedHash();
 
+    // Custom hash set via unused D3D RenderState
```

Comment:
Since remixHashFromD3D is just a part of the materialData anyways, this code should probably just live in `materialData.updateCachedHash()`